MediumDifficultyProgram6/RevString/RevString.java

```diff
@@ -0,0 +1,27 @@
+package RevString;
+
+public class RevString {
+    /*
+    Q6. Write a program to reverse the string without changing the position of special character.
+ */
+    public static void main(String[] args) {
+        String result = Reverse("Ro@hitshetty@");
```

Comment:
Please check the result for this input, char's are getting replaced with special character

Reply:
Yes, Changed code as per the comment